### DISCUSSION TITLE
How to use the Render Jinja on Table

### DISCUSSION THREAD

**Discussion by bobotdls** [frappe/print_designer#313] [General]
How to use the Render Jinja on Table

**Comment by batonac** [comment]
The column editor allows entering custom Jinja texts. Use `row` to reference values from the current row object:
<img width="1145" height="428" alt="image" src="https://github.com/user-attachments/assets/73750115-dd38-49c3-9d77-58840fcf99d2" />
<img width="665" height="335" alt="image" src="https://github.com/user-attachments/assets/ccc1624e-f207-4ac8-8218-303b36bd2d25" />

**Reply by jonaspm** [reply]
This should be added to the documentation. I spend an hour looking for this!
